src/components/storage/storageSrv.js

```diff
@@ -36,7 +36,9 @@
                     variables: {
                         uid: null
                     },
-                    cacheRules: []
+                    cacheRules: [function (path) {
+                        return path.indexOf('liveSesson') > -1;
```

Comment:
It can't be here !!!!!

this is infra global for all. You have to do it where you create the storagesrv instance for you